manifests/service.pp

```diff
@@ -6,11 +6,23 @@
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  service { 'jenkins':
-    ensure     => $jenkins::service_ensure,
-    enable     => $jenkins::service_enable,
-    hasstatus  => true,
-    hasrestart => true,
+  case $::osfamily  {
+    'OpenBSD': {
+        service { 'jenkins':
+          ensure     => $jenkins::service_ensure,
+          enable     => $jenkins::service_enable,
+          flags      => $jenkins::service_flags,
```

Comment:
I had run into trouble with a different module I made aware of those flags. Short after merge upstream, someone who ran an older version of PE complained. That version he ran, I don't remember, wasn't aware of the flags parameterr to the service, and bailed out. Therefore the two cases here, If you don't care, I can have it without the case statement ;)